internal/dockercompose/client.go

```diff
@@ -115,7 +115,9 @@ func (c *cmdDCClient) Up(ctx context.Context, spec model.DockerComposeUpSpec, sh
 	c.mu.Lock()
 	defer c.mu.Unlock()
 	runArgs := append([]string{}, genArgs...)
-	runArgs = append(runArgs, "up", "--no-deps", "--no-build", "-d")
+	// Omit --no-build for now to get v2 working.
```

Comment:
can we check the docker compose version here and use --no-build in the v1 case?

Reply:
Yes, I'll get that in here before preparing to merge.